omod/src/main/webapp/resources/scripts/mergepatients/patientSearchAndSelectWidget.js

```diff
@@ -0,0 +1,524 @@
+function PatientSearchWidget(configuration){
```

Comment:
yes, theres only little modification to allow custom handling of row selecting, and most of code is redundant. Should i resign of using modificated widget and try to write something more lightweight? or try to adapt search widget to custom handling and to be more versatile, and write roweSelectionHandlers for findPatient and mergePatient search? what do You advice?